main.py

```diff
@@ -24,11 +24,16 @@
                                             TagResourcesRequestTag
 from alibabacloud_ecs20140526.client import Client
 from alibabacloud_tea_openapi.models import Config
-
+from urllib.request import urlopen
 
 OPENSHIFT_INSTALL_GIT = "https://github.com/openshift/installer"
 REDIRECTOR_URL = "https://rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com/art/storage/releases/"
 
+FIRSTRELEASE = {}
+FIRSTRELEASE['aarch64'] = 410842021120118210
+FIRSTRELEASE['ppc64le'] = 0
+FIRSTRELEASE['s390x'] = 0
+FIRSTRELEASE['x86_64'] = 0
```

Comment:
I wonder if we need a different format, since we could be checking multiple releases?

Maybe something like:

`FIRSTRELEASE['aarch64'] = {'4.10': '410842021120118210', '4.11': '411842022020718390'}`